extensions/hibernate-envers/deployment/src/main/java/io/quarkus/hibernate/envers/deployment/HibernateEnversProcessor.java

```diff
@@ -36,8 +36,8 @@ CapabilityBuildItem capability() {
     @BuildStep
     List<AdditionalJpaModelBuildItem> addJpaModelClasses() {
         return Arrays.asList(
-                new AdditionalJpaModelBuildItem(org.hibernate.envers.DefaultRevisionEntity.class),
-                new AdditionalJpaModelBuildItem(org.hibernate.envers.DefaultTrackingModifiedEntitiesRevisionEntity.class));
+                new AdditionalJpaModelBuildItem("org.hibernate.envers.DefaultRevisionEntity"),
```

Comment:
So... no tests for these?

I'd rather have a `ClassNames` constants class for other modules as well, with the corresponding test. As long as we follow the structure I suggested, that's not too hard to maintain.

Reply:
I don't expect any of the Envers integration tests to work in case there was a mismatch here.